crates/hdi/src/link.rs

```diff
@@ -8,6 +8,15 @@ use crate::prelude::*;
 #[cfg(doc)]
 pub mod examples;
 
+/// An extension to obtain a link type filter.
+/// 
+/// Allows for single link types as well as a range or vector of link types to be passed in.
+/// To include all link types - not filter out any link type - the full range operator `..`
+/// can be used: `get_links(base, .., None)`.
+/// 
+/// Refer to the `get_links` function in
+/// ]this coordinator zome](https://github.com/holochain/holochain/blob/develop/crates/test_utils/wasm/wasm_workspace/link/src/coordinator.rs)
```

Comment:
```suggestion
/// [this coordinator zome](https://github.com/holochain/holochain/blob/develop/crates/test_utils/wasm/wasm_workspace/link/src/coordinator.rs)
```

just poking around the PRs as usual lol

Reply:
thanks @Connoropolous